go.mod

```diff
@@ -101,3 +101,5 @@ require (
 	gopkg.in/fsnotify/fsnotify.v1 v1.4.7 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+tool github.com/magefile/mage
```

Comment:
why do we need this in the go.mod?

Reply:
If someone wants to use it, they can install it. We do not need this in the project as a dependency

Reply:
anyway `mage` will be installed by [makefile](https://github.com/VictoriaMetrics/victorialogs-datasource/blob/ed3eb4681abc4e8dcc31a6543485d8a8b2965b8a/Makefile#L112-L115)
So, I would like to suggest move it to `go.mod` tool. And change to `go tool mage` into makefile. But anyway will remove these changes from this PR. And if you agree, will do it in another PR.

Reply:
We do not need this tool to build the go files. We used it to sign the plugin because of the Grafana rules.